### PR TITLE
fix: update gql settlementPrice type

### PR DESCRIPTION
**Pull request by dolcalmi**
Update settlementPrice type to avoid issues with apollo client cache. This should not be a breaking change but must be tested first in staging

### CODE REVIEW

File: src/graphql/main/schema.graphql [hunk 1]
```diff
@@ -1116,7 +1127,7 @@ type Transaction {
   """
   Price in USDCENT/SETTLEMENTUNIT at time of settlement.
   """
-  settlementPrice: Price!
+  settlementPrice: SettlementPrice!
```
Comment (nicolasburtey) [inline src/graphql/main/schema.graphql:1130]: I don't think it's backward compatible. Are we using it already?

I'm not sure if there is an easy way on the backend to see what field are been used already. I think it was one of the benefits of Apollo Studio. would be relevant to know if we can create this capability back with honeycomb

File: src/graphql/admin/schema.graphql [hunk 2]
```diff
@@ -349,6 +339,17 @@ scalar SafeInt
 """
 scalar SatAmount
 
+"""
+Price amount expressed in base/offset. To calculate, use: `base / 10^offset`
+"""
+type SettlementPrice {
+  base: SafeInt!
+  currency: DisplayCurrency!
```
Comment (nicolasburtey) [inline src/graphql/admin/schema.graphql:347]: should probably also have the underlying currency, USD or BTC dpendanding of the wallet used.

**Review by nicolasburtey — COMMENTED** [review]
some questions/comments

File: src/graphql/main/schema.graphql (hunk 1 above)
Comment (dolcalmi) [reply src/graphql/main/schema.graphql:1130]: I didnt see validations in settlement price with `__typename`. also, this is a read only field.... we can merge it and test it in staging, I think it is not a problem

File: src/graphql/admin/schema.graphql (hunk 2 above)
Comment (dolcalmi) [reply src/graphql/admin/schema.graphql:347]: why? we already have `settlementCurrency`

File: src/graphql/main/schema.graphql (hunk 1 above)
Comment (nicolasburtey) [reply src/graphql/main/schema.graphql:1130]: true. we should check with the released version of the mobile app also.

**Review by nicolasburtey — APPROVED** [review]

**Comment by dolcalmi** [comment]
closed in favor of https://github.com/GaloyMoney/galoy/pull/2439/

**Pull request closed without merging**